jdaviz/configs/cubeviz/plugins/cube_listener.py

```diff
@@ -0,0 +1,150 @@
+import numpy as np
+from contextlib import contextmanager
+import sys
+import os
+import time
+
+try:
+    from strauss.sonification import Sonification
+    from strauss.sources import Events
+    from strauss.score import Score
+    from strauss.generator import Spectralizer
+    from tqdm import tqdm
+except ImportError:
+    pass
+
+MINVOL = 1/(2**15 - 1)
+
+
+@contextmanager
+def suppress_stderr():
+    with open(os.devnull, "w") as devnull:
+        old_stderr = sys.stderr
+        sys.stderr = devnull
+        try:
+            yield
+        finally:
+            sys.stderr = old_stderr
+
+
+def audify_spectrum(spec, duration, overlap=0.05, system='mono', srate=44100, fmin=40, fmax=1300,
```

Comment:
this whole function is never reached by tests - is it possible to add coverage (or if out of scope, can we create a follow-up)?

Reply:
on closer look, it seems that the existing tests added in this PR are probably never actually running on CI because of dependencies.  Can we add strauss to at least one of the runners so they do run?

Reply:
Can I add the Strauss package to `all` in `pyproject.toml` or should I create a new workflow for Strauss?

Reply:
let's discuss - it might be time to make a convention for this and do the same thing for footprints, Roman, and strauss.